Lib9c/CurrencyExtensions.cs

```diff
@@ -26,8 +41,27 @@ public static Currency Deserialize(Bencodex.Types.Dictionary serialized)
             {
                 minters = mintersAsList.Select(b => new Address(((Binary) b).ByteArray)).ToImmutableHashSet();
             }
-            
-            return new Currency((Text)serialized["ticker"], ((Binary)serialized["decimalPlaces"]).First(), minters);
+
+            if (serialized.ContainsKey("totalSupplyTrackable"))
+            {
+                if (serialized.ContainsKey("maximumSupplyMajor"))
```

Comment:
Is it okay to check `maximumSupplyMinor` also?

Reply:
If there is `maximumSupplyMajor`, there also must be `maximumSupplyMinor` as well. If one of them are missing it's a malformed serialization. Maybe should I raise an exception if one of them are missing?

Reply:
I'm sorry for confusing about the `Currency` struct. As you said, it should raise an exception because it's a malformed serialization. And I expect it will raise an exception like `KeyNotFoundException` in the below line:

```
(Integer)serialized["maximumSupplyMinor"]
```

I think this confusion is from serialization implementation was leaked from out `Currency`.

I asked a question related with this problem on Discord Server. https://discord.com/channels/928926944937013338/970554437733073027/1020254911495159808

Reply:
Added logic to reject malformed serializations.